vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -159,8 +159,6 @@ public void commit(int updateId) {
             enqueue("$connector.confirm", updateId, ComboBox.this.lastFilter);
             queue.forEach(Runnable::run);
             queue.clear();
-
-            ComboBox.this.lastFilter = null;
```

Comment:
Hey, @bogdanudrescu can you verify if it's okay to remove this line? For me it seems that it is not needed, but can you verify this as the author of this line?

Reply:
To clarify, I need to keep the reference to the last filter, because I need to compare if the filter has changed from last request or not.

I could also create another property for this purpose, but it would be weird to have two properties saving the last filter string.

Reply:
Hi @pekam, sorry for the late reply and thanks for noticing this. I think your point makes sense and we should remove that line, always keeping the value in the `lastFilter`.